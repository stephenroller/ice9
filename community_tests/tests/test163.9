# forward & 2 procs w/ same name							fail

forward foo(a,b : int) : int

proc foo (a,b : int) : int
end

proc 
foo
(a,b : int) : int
end


;