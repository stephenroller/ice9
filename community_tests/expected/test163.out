line 9: proc foo is already defined